core/domain/stats_jobs_one_off.py

```diff
@@ -259,8 +263,14 @@ def reduce(exp_id, stringified_values):
                     state_stats_mapping[state_name] = (
                         stats_domain.StateStats.create_default())
             else:
-                exp_commit_log = exp_models.ExplorationCommitLogEntryModel.get(
-                    'exploration-%s-%s' % (exp_id, version))
+                try:
+                    exp_commit_log = (
+                        exp_models.ExplorationCommitLogEntryModel.get(
+                            'exploration-%s-%s' % (exp_id, version)))
+                except Exception:
+                    yield "Check if the exploration %s has snapshot models." % (
```

Comment:
Maybe add the version number to the logging statement as well?

Also, probably better to just say what happened, e.g. "ERROR: Missing CommitLogEntryModel for exploration %s, version %s". In fact you may want to do this with the others, and prefix it with "ERROR: " so that the error logs are easy to detect among the others.

Reply:
Done.